web/tests/search_suggestion.test.js

```diff
@@ -504,6 +505,7 @@ test("check_is_suggestions", ({override, mock_template}) => {
         "-is:alerted",
         "-is:unread",
         "-is:resolved",
+        "is:resolved",
```

Comment:
why is this being included in a query that starts with a `-`?

Reply:
There was an agreement to include it in [CZO](https://chat.zulip.org/#narrow/channel/101-design/topic/searching.20for.20unresolved.20topics/near/1949024). Due to the confusion, I'm wondering if we should change course and just remove it entirely.